src/main/java/com/example/TaskFlow/services/UserServiceImpl.java

```diff
@@ -28,7 +28,7 @@ public UserDTO createUser(UserDTO userDTO) {
         User userEntitySave = objectMapper.convertValue(userDTO, User.class);
         userValidationService.firstNameValidation(userDTO.getFirstName());
         userValidationService.lastNAmeValidation(userDTO.getLastName());
-        userValidationService.emailValidation(userDTO.getEmail());
+        userValidationService.validateEmail(userDTO.getEmail());
         User userResponseEntity = userRepository.save(userEntitySave);
         log.info("User created with id: {}", userResponseEntity.getId());
         return objectMapper.convertValue(userResponseEntity, UserDTO.class);
```

Comment:
Try to add some space between the lines of this code.

```suggestion
        User userEntitySave = objectMapper.convertValue(userDTO, User.class);
        userValidationService.firstNameValidation(userDTO.getFirstName());
        userValidationService.lastNAmeValidation(userDTO.getLastName());
        userValidationService.validateEmail(userDTO.getEmail());
        
        User userResponseEntity = userRepository.save(userEntitySave);
        log.info("User created with id: {}", userResponseEntity.getId());
        
        return objectMapper.convertValue(userResponseEntity, UserDTO.class);
```